astronomer/providers/apache/hive/example_dags/example_hive.py

```diff
@@ -0,0 +1,23 @@
+"""This is an example dag for hive partition sensors."""
+from datetime import datetime
+
+from airflow import DAG
+
+from astronomer.providers.apache.hive.sensors.hive_partition import (
+    HivePartitionSensorAsync,
+)
+
+with DAG(
```

Comment:
you will have to add this in integration test too

Reply:
and also make the integration DAG self sufficient (SSH tunneling would be required, so `LivyOperatorAsync` example DAG may be a good one to refer)

Reply:
Done